tests/Unit/PointwiseFunctions/EquationsOfState/TestHelpers.hpp

```diff
@@ -90,6 +90,30 @@ void check_impl(const std::unique_ptr<::EquationsOfState::EquationOfState<
         [](const auto& /*meta*/, const auto& /*meta*/) {})(
         std::integral_constant<size_t, ThermodynamicDim>{},
         std::add_pointer_t<EoS>{nullptr});
+    INFO("Done\nTesting circular computation...")
```

Comment:
This is not useful info. `Testing that rest_mass_density_from_enthalpy and specific_enthalpy_from_density are inverses of each other`.